docs/resources/remote.md

```diff
@@ -79,6 +79,9 @@ All generic repo arguments are supported, in addition to:
   * `statistics_enabled` - (Optional, Default: false) If set, Artifactory will notify the remote instance whenever an artifact in the Smart Remote Repository is downloaded locally so that it can update its download counter. Note that if this option is not set, there may be a discrepancy between the number of artifacts reported to have been downloaded in the different Artifactory instances of the proxy chain.
   * `properties_enabled` - (Optional, Default: false) If set, properties for artifacts that have been cached in this repository will be updated if they are modified in the artifact hosted at the remote Artifactory instance. The trigger to synchronize the properties is download of the artifact from the remote repository cache of the local Artifactory instance.
   * `source_origin_absence_detection` - (Optional, Default: false) If set, Artifactory displays an indication on cached items if they have been deleted from the corresponding repository in the remote Artifactory instance.
+
+~>`propagate_query_params` is only available for Generic type repositories.
```

Comment:
No, it will be allowed to set it with the remote repo resource, but in case of `gems` the user won't be able to use it as a source, so it will be broken.
For now, the documentation will have a note, later we will think about how to restrict users from doing that. 
In the UI you can't set `Propagate Query Params` for any other repo than Generic Remote, but API allows you to do it.